scripts/install/autobrr.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# autobrr installer
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_reqs=("curl" "sqlite3")
+apt_install "${app_reqs[@]}"
+
+users=($(_get_user_list))
+
+if [[ -n $1 ]]; then
+    user=$1
+    _autobrr_user_config ${user}
+    if [[ -f /install/.nginx.lock ]]; then
+        echo_progress_start "Configuring nginx"
+        bash /etc/swizzin/scripts/nginx/autobrr.sh
+        systemctl reload nginx
+        echo_progress_done
+    fi
+    exit 0
+fi
+
+_install_autobrr() {
+
+    echo_progress_start "Downloading release archive"
+
+    case "$(_os_arch)" in
+        "amd64") arch='x86_64' ;;
+        "arm64") arch="arm64" ;;
+        "armhf") arch="armv6" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    latest=$(curl -sL https://api.github.com/repos/autobrr/autobrr/releases/latest | grep "linux_$arch" | grep browser_download_url | cut -d \" -f4) || {
+        echo_error "Failed to query GitHub for latest version"
+        exit 1
+    }
+
+    if ! curl "$latest" -L -o "/tmp/autobrr.tar.gz" >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Archive downloaded"
+
+    echo_progress_start "Extracting archive"
+
+    # the archive contains both autobrr and autobrrctl to easily setup the user
+    tar xfv "/tmp/autobrr.tar.gz" --directory /usr/bin/ >> "$log" 2>&1 || {
```

Comment:
no need to ensure perms to `+x`? if the perms are stored in the archive then it won't be an issue, just curious

Reply:
The packager handles all that so the binaries will always be executable.